landings/a/index.liquid

```diff
@@ -164,18 +164,24 @@ document.addEventListener('DOMContentLoaded', function() {
 	});
 
 	let telegram = document.getElementById("telegram");
-    telegram.addEventListener('click', () => {
-		let w = window.open('_blank');
-        w.location = url;
-        window.amplitude.track('landing:a:click-telegram');
-    });
-
-    let whatsapp = document.getElementById("whatsapp");
-    whatsapp.addEventListener('click', () => {
-		let w = window.open('_blank');
-        w.location = url;
-        window.amplitude.track('landing:a:click-whatsapp');
-    });
+	telegram.addEventListener('click', (event) => {
+		event.preventDefault();
+		let redirectUrl = new URL('https://t.me/+573107887042');
+		amplitude.track('landing:a:click-telegram')
+		.then(() => {
+			window.open(redirectUrl.toString(), '_blank');
```

Comment:
this will not works on mobile, you have to open first the window, later do the track and change the location for the window:

```
let redirectUrl = new URL('https://t.me/+573107887042');
const newWindow = window.open('_blank');
amplitude.track('landing:a:click-telegram')
		.then(() => {
			newWindow.location = redirectUrl.toString();
		})

```